conda_smithy/anaconda_token_rotation.py

```diff
@@ -361,9 +338,10 @@ def rotate_token_in_travis(
 
 
 def rotate_token_in_azure(user, project, binstar_token, token_name):
+    from vsts.build.v4_1.models import BuildDefinitionVariable
+
     from .azure_ci_utils import build_client, get_default_build_definition
     from .azure_ci_utils import default_config as config
-    from vsts.build.v4_1.models import BuildDefinitionVariable
```

Comment:
Out of curiosity, do we know what led to this change?

Reply:
The ruff linter `I` rules sort imports with `isort`. Does that answer your question?